dev/gen-anvil-state.sh

```
@@ -0,0 +1,80 @@
+      --arg identity_update_broadcaster "$identity_update_broadcaster_address" \
+      --arg node_registry "$node_registry_address" \
+      '{
+        create3_factory_address: $create3_factory,
+        rate_registry_address: $rate_registry,
+        message_group_broadcaster_address: $message_group_broadcaster,
+        identity_update_broadcaster_address: $identity_update_broadcaster,
+        node_registry_address: $node_registry
+      }' > "${anvil_state_info_file}"
+}
+
+if [ ! command -v anvil &> /dev/null ]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)